app/helpers/metadata_helper.rb

```diff
@@ -86,9 +86,9 @@ module MetadataHelper
   IDENTIFIER = RDF::URI(DC_TERMS_BASE_URI + 'identifier') unless const_defined?(:IDENTIFIER)
   SOURCE     = RDF::URI(DC_TERMS_BASE_URI + 'source') unless const_defined?(:SOURCE)
   TITLE      = RDF::URI(DC_TERMS_BASE_URI + 'title') unless const_defined?(:TITLE)
-  RIGHTS      = RDF::URI(DC_TERMS_BASE_URI + 'rights') unless const_defined?(:RIGHTS)
-  DESCRIPTION = RDF::URI(DC_TERMS_BASE_URI + 'description') unless const_defined?(:DESCRIPTION)
-  BIBLIOGRAPHIC_CITATION = RDF::URI(DC_TERMS_BASE_URI + 'bibliographicCitation') unless const_defined?(:BIBLIO_CITATION)
+  RIGHTS      = RDF::URI(DC_ELEMENTS_BASE_URI + 'rights') unless const_defined?(:RIGHTS)
+  DESCRIPTION = RDF::URI(DC_ELEMENTS_BASE_URI + 'description') unless const_defined?(:DESCRIPTION)
+  BIBLIOGRAPHIC_CITATION = RDF::URI(DC_ELEMENTS_BASE_URI + 'bibliographicCitation') unless const_defined?(:BIBLIO_CITATION)
```

Comment:
I think `BIBLIOGRAPHIC_CITATION` is a [term](http://dublincore.org/documents/dcmi-terms/), not an [element](http://dublincore.org/documents/dces/)